packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -670,16 +667,6 @@ export class RunningSummarizer implements IDisposable {
 					}
 				}
 
-				// If all attempts failed, log error (with last attempt info) and close the summarizer container
-				this.mc.logger.sendErrorEvent(
```

Comment:
the PR description mentions that we want these to be generic, but this removes the event altogether.

Reply:
Oh, I was removing this event, as it is not that useful. I asked around, no one looks at the `FailToSummarize` event, rather they look at the last `summarize_cancel` event.

Reply:
Are they always 1:1 in telemetry?  And is there any info contained in here (e.g. `reason`, the `message`/`error` on `lastResult`) that isn't logged elsewhere?

Reply:
This event does not contain anything that isn't already in the `summarize_cancel` event.